src/klicky.ts

```diff
@@ -1,3 +1,54 @@
 export const Klicky = () => {
   console.log("klickly ran!");
 };
+
+const outer = document.querySelector('.outer');
+
+const capture = {
+  capture : true
+};
+
+const noneCapture = {
+  capture : false 
+};
+
+const once = {
+  once : false 
+};
+
+outer?.addEventListener('click', onceHandler, once);
+
+function onceHandler(event)
+{
+  console.log("You have clicked the page");
+  write();
+  var date = new Date();
+  dateTime(date);
+}
+
+function write()
+{
+  const para = document.createElement("p");
+  para.innerText = "You have clicked the page!";
+  document.body.appendChild(para);
+
+  var dt = new Date();
+  const datetime = document.createElement("p");
+  datetime.innerHTML = dt;
+  document.body.appendChild(datetime);
+
+  document.getElementById('date-time').innerHTML = dt;
+}
+
+function dateTime(date:Date):void
+{
+  var month = date.getMonth()+1;
+  var day = date.getDate();
+  var year = date.getFullYear();
+
+  var hours = date.getHours();
+  var minutes = date.getMinutes();
+  var seconds = date.getSeconds();
+
+  console.log("Click event occurred at: "+month+"/"+day+"/"+year+" "+hours+":"+minutes+":"+seconds); 
```

Comment:
nice job making this console log! This is fine to leave as is, but isn't it annoying to have to do all this string concatenation? Try looking up "template literal" syntax to see if you can make this a bit cleaner